pre.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euo pipefail
+source ${GITHUB_ACTION_PATH:-.}/common.sh
+
+echo "------------------------------------------------------------------------------"
+if [[ (-d ${RELEASE_PRE_SCRIPTS:-}) && (${MAVEN_RELEASE_EXECUTE_PRE_SCRIPTS} == "true") ]]
```

Comment:
does it work for empty `RELEASE_PRE_SCRIPTS`? You are doing `-d` test for empty string.